scripts/lint-dividing-lines.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if grep -Ernxo -e "/{4,79}" -e "/{81,}" ./src/; then
```

Comment:
Have you considered making this a test in `kamu-repo-tools`, just like the one that checks for license headers? This is nice an concise but I generally prefer to avoid shell scripts due to slight differences in utils version and portability.

Reply:
No problem, will be done

Reply:
Done

Reply:
Added https://github.com/kamu-data/kamu-cli/pull/685/files#diff-ac447473973f0a9697d6262d62ac648b6563dfffdb827e2bd7531dee2f0b8932